index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <span class="logo-nombre">
+            Gustavo Porcilio
+        </span>
+        <div class="hamburger">
+            <nav class="navlinks">
+                <a href="#porfolio">PORFOLIO</a>
+                <a href="#about">ABOUT</a>
+                <a href="#contact">CONTACT</a>
+            </nav>
+        </div>
+
+    </header>
+    <section class="principal">
+        <div>
+            <h1 class="tittle">Hey There! I'm Gustavo</h1>
+        </div>
+        <div class="profession">
+            <h2>I'm a software developer</h2>
+        </div>
+        <div class="text">
+            <p class="presentation">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="icons">
+            <a href="https://github.com/MarcoAurelioAntonio" class="github"></a>
+            <a href="https://twitter.com/gustavoporcilio" class="twitter"></a>
+            <a href="https://www.linkedin.com/in/gustavo-porcilio-4496a223a/" class="linkedin"></a>
+        </div>
```

Comment:
- [ ] Kindly make sure the list of icons uses a `<ul>` tag instead of a `<div>` to group them, and `<li>` per icon; so then each icon has a link (`a` tag). i.e:
```
      <ul>
            <li><a href="https://github.com/MarcoAurelioAntonio" class="github"></a></li>
            ...
        </ul>
```

- [ ] Also make sure you add the other icons to be 5 as on the Figma template and if you don't have accounts yet you can use for now `<a href="#" ></a>` a **#** on the `href` attribute.